firebase-local/rtdb.rules.json

```diff
@@ -1,5 +1,8 @@
 {
```

Comment:
Refactored the rules a bit to just give admins global access, and we will need to copy/paste this into Firebase. I am doing so in the test environment now